index.html

```diff
@@ -148,8 +148,28 @@ <h3>Entry Requirements</h3>
           </p>
         </div>
       </div>
+<!-- Bug Board -->
+  <div class="ui vertical stripe quote segment">
+    <div class="ui equal width stackable internally celled grid">
+      <div class="left aligned row">
+        <div class="column">
+          <h3>Fix A Bug</h3>
+          <p>
+            As you can see we have many bugs to fix on our site. If you would like to
+            help with that take a look at our <a href="https://trello.com/b/CmNMjNHc/
+            quick-main-website-projects">Bug Board!</a><br></p>
+            <p>Come and find or GitHub repo, clone, and contribute!
```

Comment:
Starting from this `<p`> tag, and going down to the fourth closing `</div` tag, I recommended reducing the indentation of all the lines by one level. Then everything lines up nicely and it's easier to see that you've correctly closed all four `div`s!